Source/DataSources/KmlTour.js

```diff
@@ -81,14 +82,42 @@ KmlTour.prototype.addPlaylistEntry = function (entry) {
   this.playlist.push(entry);
 };
 
+/**
+ * Play this tour.
+ *
+ * @param {CesiumWidget} widget The Cesium widget.
+ * @param {Object} [cameraOptions] these options will be merged with {@link Camera#flyTo}
+ * options for FlyTo playlist entries.
+ */
+KmlTour.prototype.playInWidget = function (widget, cameraOptions) {
+  this.tourStart.raiseEvent();
+
+  const tour = this;
+  playEntry.call(this, widget, cameraOptions, function (terminated) {
+    tour.playlistIndex = 0;
+    // Stop nonblocking entries
+    if (!terminated) {
+      cancelAllEntries(tour._activeEntries);
+    }
+    tour.tourEnd.raiseEvent(terminated);
+  });
+};
+
 /**
  * Play this tour.
  *
  * @param {Viewer} viewer viewer widget.
  * @param {Object} [cameraOptions] these options will be merged with {@link Camera#flyTo}
  * options for FlyTo playlist entries.
+ *
+ * @deprecated
  */
 KmlTour.prototype.play = function (viewer, cameraOptions) {
```

Comment:
The Sandcastle `KML Tours.html` will also need to be updated to use the new API.

Reply:
Updated.